patches/chromium/remove_usage_of_incognito_apis_in_the_spellchecker.patch

```diff
@@ -0,0 +1,21 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Samuel Attard <sattard@slack-corp.com>
+Date: Wed, 23 Oct 2019 11:43:58 -0700
+Subject: remove usage of incognito APIs in the spellchecker
+
```

Comment:
Can you explain the why behind this patch and if/when we can potentially remove it? (for posterity)

Reply:
Explained